plugins/catalog/src/components/AboutCard/AboutContent.tsx

```diff
@@ -14,207 +14,42 @@
-
-const useStyles = makeStyles({
-  description: {
-    wordBreak: 'break-word',
-  },
-});
+import { AboutCardDefaultFields } from './fields';
 
 /**
  * Props for {@link AboutContent}.
  *
  * @public
  */
 export interface AboutContentProps {
-  entity: Entity;
-}
-
-function getLocationTargetHref(
-  target: string,
-  type: string,
-  entitySourceLocation: {
-    type: string;
-    target: string;
-  },
-): string {
-  if (type === 'url' || target.includes('://')) {
-    return target;
-  }
-
-  const srcLocationUrl =
-    entitySourceLocation.type === 'file'
-      ? `file://${entitySourceLocation.target}`
-      : entitySourceLocation.target;
-
-  if (type === 'file' || entitySourceLocation.type === 'file') {
-    return new URL(target, srcLocationUrl).href;
-  }
-
-  return srcLocationUrl;
+  /**
+   * The entity here is for backward compatibility only.
+   * It's not passed to the children as the children use
+   * useEntity hook to access the entity in the current context.
+   *
+   * The {@link AboutContent} component should only be used
+   * within EntityLayout, and the entity prop will be removed
+   * in future releases.
+   *
+   * @deprecated use useEntity hook in children.
+   */
+  entity?: Entity;
+  children?: React.ReactNode;
 }
 
+// TODO: It seems the whole AboutContent component here is redundant,
+// consider deprecating and removing it.
 /** @public */
-export function AboutContent(props: AboutContentProps) {
-  const { entity } = props;
-  const classes = useStyles();
-  const isSystem = entity.kind.toLocaleLowerCase('en-US') === 'system';
-  const isResource = entity.kind.toLocaleLowerCase('en-US') === 'resource';
-  const isComponent = entity.kind.toLocaleLowerCase('en-US') === 'component';
-  const isAPI = entity.kind.toLocaleLowerCase('en-US') === 'api';
-  const isTemplate = entity.kind.toLocaleLowerCase('en-US') === 'template';
-  const isLocation = entity.kind.toLocaleLowerCase('en-US') === 'location';
-  const isGroup = entity.kind.toLocaleLowerCase('en-US') === 'group';
-
-  const partOfSystemRelations = getEntityRelations(entity, RELATION_PART_OF, {
-    kind: 'system',
-  });
-  const partOfComponentRelations = getEntityRelations(
-    entity,
-    RELATION_PART_OF,
-    {
-      kind: 'component',
-    },
-  );
-  const partOfDomainRelations = getEntityRelations(entity, RELATION_PART_OF, {
-    kind: 'domain',
-  });
-  const ownedByRelations = getEntityRelations(entity, RELATION_OWNED_BY);
-
-  let entitySourceLocation:
-    | {
-        type: string;
-        target: string;
-      }
-    | undefined;
-  try {
-    entitySourceLocation = getEntitySourceLocation(entity);
-  } catch (e) {
-    entitySourceLocation = undefined;
+export const AboutContent = (props: AboutContentProps) => {
+  const { entity, children } = props;
+  if (entity !== undefined) {
+    // eslint-disable-next-line no-console
+    console.warn(
+      'Make sure to use this component in an EntityLayout or inside an EntityProvider',
```

Comment:
Wait, won't this log only if you ARE inside a provider? 🤔 Besides, probably best to not log at all anyway; anybody that tries to use useEntity inside will get a warning in either case

Reply:
It only logs when the entity prop is present, which means the caller of `AboutContent` passes in the entity prop. The intention here is to remind users that the entity prop won't be passed on to children since they will use `useEntity` hook to access the entity. Is there a better way to address this?

Reply:
>  Only thing maybe then is to only do it once so we don't spam the logs too much.

Make sense, but I'm not sure how to log only once, should I use [lodash.once](https://lodash.com/docs/4.17.15#once) for this?

Reply:
Sure, if you want, but remember to make that function outside of the component then. May also want to add the word `AboutContent` to the message so people know which particular usage to address